numpy/core/src/multiarray/calculation.c

```diff
@@ -423,7 +423,7 @@ __New_PyArray_Std(PyArrayObject *self, int axis, int rtype, PyArrayObject *out,
         return NULL;
     }
     arr2 = (PyArrayObject *)PyArray_EnsureAnyArray(
-                PyArray_GenericBinaryFunction(arr1, obj3, n_ops.multiply));
+            PyArray_GenericBinaryFunction((PyObject *)arr1, obj3, n_ops.multiply));
```

Comment:
this dedent seems wrong
```suggestion
                PyArray_GenericBinaryFunction((PyObject *)arr1, obj3,
                                               n_ops.multiply));
```